Communitheme/gtk-3.0/_common.scss

```diff
@@ -4513,10 +4513,11 @@ decoration {
   // server-side decorations as used by mutter
   .ssd & { box-shadow: none; }
 
+  //These shadows are applied to the context-menu!
   .csd.popup & {
     border-radius: $small_radius;
-    box-shadow: 0 1px 2px transparentize(black, 0.8),
-                0 0 0 1px transparentize($_wm_border, 0.1);
+    box-shadow: 0 2px 5px transparentize(black, 0.75),
+    0 0 0 1px transparentize($_wm_border, 0.1);
```

Comment:
Could you keep the same alignment we had between the two box-shadows?

Reply:
Done, thanks - should drink a coffee!